tests/unit/test_federation_policy

```diff
@@ -0,0 +1,84 @@
+#!/bin/bash
```

Comment:
Well, this is interesting. Instead of doing this single unit test, shouldn't it run all unit tests? I can probably find other cases of tests that would be easier to do from bash like this than how some of the policy acceptance tests are done

Reply:
I added test_federation_policy to Makefile.am so would expect it to get run: https://github.com/cfengine/masterfiles/pull/1472/files#diff-156d4744b2414baf39a261e75355322eR8

Reply:
right, i saw that, it just made me think how we need general bash tests, no framework really exists in our stuff for bash tests today